src/update-changelog.ts

```diff
@@ -213,10 +213,12 @@ async function runCommand(command: string, args: string[]): Promise<string[]> {
  * @param description - The commit message description.
  * @returns The category of the change.
  */
-function getCategory(description: string): ChangeCategory {
-  // Check if description contains a colon
-  if (description.includes(':')) {
-    const [prefix] = description.split(':').map((part) => part.trim());
+export function getCategory(description: string): ChangeCategory {
+  const conventionalCommitPattern = /^(feat|fix)(?:\([^)]*\))?\s*:\s*/u;
```

Comment:
nit: An empty scope string should probably be considered to be invalid.

```suggestion
  const conventionalCommitPattern = /^(feat|fix)(?:\([^)]+\))?\s*:\s*/u;
```